--- libs-0.13.4/driver/syscall_table64.c.orig	2023-10-26 18:38:45.000000000 +0200
+++ libs-0.13.4/driver/syscall_table64.c	2023-11-19 10:03:55.040625438 +0100
@@ -23,8 +23,10 @@ or GPL2.txt for full copies of the licen
 #include "syscall_compat_aarch64.h"
 #elif defined(__s390x__)
 #include "syscall_compat_s390x.h"
+#else // fake
+#include "syscall_compat_x86_64.h"
 #endif /* __x86_64__ */
-#elif defined(_MSC_VER) || defined(__EMSCRIPTEN__)
+#else
 // these are Linux syscall numbers and obviously meaningless for Windows/macOS
 // but we need *some* definition so that we have a mapping for scap_ppm_sc.c
 #include "syscall_compat_x86_64.h"
